game-core/src/main/java/games/strategy/engine/posted/game/pbf/ForumPostingException.java

```diff
@@ -0,0 +1,8 @@
+package games.strategy.engine.posted.game.pbf;
+
+public class ForumPostingException extends RuntimeException {
```

Comment:
Codacy found an issue: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=5774013)